deployment/ansible/roles/raster-foundry.jenkins/tasks/main.yml

```diff
@@ -58,3 +58,30 @@
     job: docker system prune -af
     state: "present"
   changed_when: False
+
+- name: Setup weekly NPM cleanup cron task
+  cron:
+    name: "npm-cleanup"
+    user: "{{ jenkins_name }}"
+    special_time: "weekly"
+    job: find /var/lib/jenkins/workspace/ -path "*/.node_modules/*" -delete
+    state: "present"
+  changed_when: False
+
+- name: Setup weekly Maven cleanup cron task
+  cron:
+    name: "maven-cleanup"
+    user: "{{ jenkins_name }}"
```

Comment:
Use root user to get around file permission issues when files are written to the host file system through the context of a container.